internal/api/v1alpha1/authproxyworkload_types.go

```diff
@@ -182,6 +182,16 @@ type TelemetrySpec struct {
 	// This sets the proxy container's CLI argument `--http-port`
 	//+kubebuilder:validation:Optional
 	HTTPPort *int32 `json:"httpPort,omitempty"`
+
+	// AdminPort the port for the /quitquitquit endpoint.
```

Comment:
I guess this is the string that needs to change.

Reply:
Fixed.